interface/globals.php

```diff
@@ -465,4 +465,23 @@ function strterm($string,$length) {
   extract($_GET,EXTR_SKIP);
   extract($_POST,EXTR_SKIP);
 }
+//Create page validation array -- all the pages that have to fire validatejs and their rules
+function collectValidationPageRules($option_id,$title){
+    if (!empty($_SESSION['authUserID'])) {
+
+
+        $pages = sqlStatement("SELECT * " .
+            "FROM `list_options` WHERE list_id='PageValidation' AND option_id='$option_id' AND title='$title'");
+
+          $row = sqlFetchArray($pages);
+          if($row) {
+              return array('page_name' => $row['option_id'] . ".php", 'rules' => $row['notes']);
+          }
+          else{
+              return null;
+          }
+
```

Comment:
this function would be more clear with the sqlQuery function since just grabbing 1 value (note the list_id, option_id combo can only contain 1 entry):
``````
$pages = sqlQuery("SELECT notes FROM list_options WHERE list_id = 'PageValidation' AND option_id = ?", array($option_id));
if (!empty($pages['notes'])) {
  return array('rules' => $pages['notes']);
else {
  return null;
}